docs/README_KOKKOS.md

```diff
@@ -349,7 +349,7 @@ units metal
 atom_style atomic/kk
 read_data silicon.data
 run_style verlet/kk
-pair_style metatensor/kk pet-mad-latest.pt device cuda extensions extensions/
+pair_style metatensor/kk pet-mad-latest.pt device cuda extensions extensions/ check_consistency off
```

Comment:
Maybe adding some new lines for better reading. As we do it in the general example?

```suggestion
pair_style metatensor/kk &
   pet-mad-latest.pt &
   device cuda &
   extensions extensions/ &
```

I think we can also just remove the lines with `check_consistency off`